site/docs/reference/Connectors/capture-connectors/README.md

```diff
@@ -48,22 +48,22 @@ All Estuary connectors capture data in real time, as it appears in the source sy
   * [Configuration](./http-ingest.md)
   * Package - ghcr.io/estuary/source-http-ingest:dev
 * MariaDB
-  * [Configuration](./mariadb.md)
+  * [Configuration](./MariaDB/)
```

Comment:
The paths for a couple of these connector docs changed when I changed the folder structure. This could have been solved with a rebase or by keeping master up to date

Reply:
Fixed by merging commit  [here](179912a1c8e35acfe918fac2b547628d3136ba7e)